lighthouse-core/test/results/sample_v2.json

```diff
@@ -5,7 +5,7 @@
     "hostUserAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3358.0 Safari/537.36",
     "benchmarkIndex": 1000
   },
-  "lighthouseVersion": "4.1.0",
+  "lighthouseVersion": "4.1.0+cli",
```

Comment:
I don't know if I like an addition to the version itself, since it really _isn't_ part of the version.  I like it better as it's own item in `channel` only.

Reply:
Enough of the channels have had their own cherry-picks though that I think one *could* make the argument that it has effectively become a subversion. Not sure if I feel strongly enough to make that argument myself, but one could :)

Reply:
I don't know why we'd need it to be part of the version. The [old PR](https://github.com/GoogleChrome/lighthouse/pull/4972) mentioned there was a decision to include it, and @paulirish [made a comment](https://github.com/GoogleChrome/lighthouse/pull/4972#issuecomment-384365102) that seems like a summary of some kind of decision.

Reply:
Back then there was no `channel` value in the settings, so maybe all we wanted is to include the channel in the LHR somehow?

Reply:
yeah, i would definitely prefer a separate `channel` entry. I believe @paulirish may have changed his thinking on this as well.

I don't remember what the earlier discussion was about, and there isn't any mention in old meeting notes that I can find, so let's call that moot :)